app/Console/Commands/CreateCSVCommand.php

```diff
@@ -0,0 +1,82 @@
+<?php
```

Comment:
Optional command to create a test CSV file:
```
php artisan create:csv
```
Files will be created in a new csvFiles folder in project's root.